apps/web/components/booking/BookingListItem.tsx

```diff
@@ -444,6 +444,7 @@ function BookingListItem(booking: BookingItemProps) {
           setIsOpenDialog={setIsOpenReassignDialog}
           bookingId={booking.id}
           teamId={booking.eventType?.team?.id || 0}
+          bookingFromRoutingForm={isBookingReroutable(parsedBooking)}
```

Comment:
We're using the same variable when determine if the reroute option should be assigned.